pkg/mount-manager/safe-mounter_windows.go

```diff
@@ -22,16 +22,18 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
-	diskv1alpha1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1alpha1"
-	diskclientv1alpha1 "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1alpha1"
+	diskv1beta1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta1"
```

Comment:
if we alias it to some common name, like "diskapi", then there's potentially less lines that need changing if we have to change api versions again

Reply:
updated. thanks!